### ISSUE TITLE
Shelter 2013 - Help - Image URL not working 

### ISSUE THREAD

**Issue by tagapic** [geotagx/geotagx-theme#135]
**Is the building or shelter affected by flood waters?**
Just noticed that one of the images referred to in the "Help" to the question does not work any more.
See URL below.
http://columbian.media.clients.ellingtoncms.com/img/photos/2013/06/19/India_Floods.JPEG-08b00_r770x495.jpg?e13abf78c5373145db895a72bc651d1d4e1d8b6f

**Suggest** also improving the help **text** and adding more images here to better illustrate and distinguish dwellings that are "inundated" and those that have been "washed away".

**Comment by ValeErre87** [comment]
@tagapic thank you for pointing this out. We will work on the image that has been delete, to see if we can find it again or change it with another one.

Also, thanks for the comment about the text. What do you mean exactly by "improving the text"? Do you think that it would be enough to add more image or it would be better to rephrase it?

**Comment by tagapic** [comment]
Thanks for looking into this. By improving the text I meant rephrasing to explain the difference between an "inundated" building/shelter from a "washed away" one. For example, the text could include the signs to look for in classifying as one or the other choice. Also suggest referring to the same word "words" in the Options text as in the Question and Help text when referring to structures. There are three words - building, shelter, dwelling - that are used and these need to be distinguished. 

**Shelter** is used more in reference to temporary homes - tents, refugee camps, a school or church used to house people temporarily etc. and as such would be better used to classify temporary dwellings.  
**Dwelling** is usually referred to a private home, residence, house or apartment. They would include Pucca, Semi-pucca and Kutcha (meaning raw) structures i.e. jhuggis and other hut like structures.
**Building** suggests a pucca or semi-pucca structure - could be residential or for commercial purpose.

The **image** used in the general tutorial could be used to show [inundated dwellings/buildings](http://www.sify.com/news/yamuna-swells-flood-threat-increases-in-delhi-news-national-kjwn4caeige.html) and perhaps replace the broken link. Additional images could be that of inundated buildings (apartment complex, village, market)... or of a washed away home or building.

Some images show **shelter/s** - usually tents - either constructed by authorities or by community (using tent materials, cots and other barricades) following the washing away of their homes by the flood. 

Hope I am clarifying better. thanks.

**Comment by ValeErre87** [comment]
@tagapic yes, this is very clear. It is a very good point, and we will definitely consider to change the help text to make it clearer.

I am keeping track of all your suggestions (including the other topics you are opening) to address them at our earliest.
Thank you very much for your contributions.
